src/Sentry/SpanTracer.cs

```diff
@@ -82,13 +82,21 @@ public void SetTag(string key, string value) =>
     public void UnsetTag(string key) =>
         (InternalTags ??= new ConcurrentDictionary<string, string>()).TryRemove(key, out _);
 
-    private readonly ConcurrentDictionary<string, object?> _data = new();
+    private Dictionary<string, object?>? _data;
 
     /// <inheritdoc />
-    public IReadOnlyDictionary<string, object?> Extra => _data;
+    public IReadOnlyDictionary<string, object?> Data =>
```

Comment:
This will be accessed concurrently so needs synchronization. Keep the original implementation of map?

Reply:
Yup - that's what I get for mass copy/pasting on this one.  Thanks for the spot.